src/Symfony/Component/Config/Tests/Resource/FileExistenceResourceTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config\Tests\Resource;
+
+use Symfony\Component\Config\Resource\FileExistenceResource;
+
+class FileExistenceResourceTest extends \PHPUnit_Framework_TestCase
+{
+    protected $resource;
+    protected $file;
+    protected $time;
+
+    protected function setUp()
+    {
+        $this->file = realpath(sys_get_temp_dir()).'/tmp.xml';
+        $this->time = time();
+        $this->resource = new FileExistenceResource($this->file);
+    }
+
+    protected function tearDown()
+    {
+        if (file_exists($this->file)) {
+            unlink($this->file);
+        }
+    }
+
+    public function testToString()
+    {
+        $this->assertSame($this->file, (string) $this->resource);
+    }
+
+    public function testGetResource()
+    {
+        $this->assertSame($this->file, $this->resource->getResource(), '->getResource() returns the path to the resource');
+    }
+
+    public function testIsFreshWithExistingResource()
```

Comment:
IMO this test is not concise in testing the relevant things.

I'd do the following steps:
- Touch the file
- Create the `FileExistenceResource` for the file, serialize it.
- Unserialize it, assert `isFresh` returns true
- Remove the file
- Unserialize resource, assert `isFresh` returns false

Then repeat with a nonexistent file that is created later on.

(My $10^-2)